Tests/Resources/Fixtures/config/usernameMultipleFailure.yml

```diff
@@ -0,0 +1,32 @@
+doctrine_phpcr:
+    session:
+        sessions:
+            default:
+                backend:
+                    type: jackrabbit
+                    url: http://a
+                workspace: "default"
+                username: "admin"
+                password: "admin"
+
+            website:
+                backend:
+                    type: jackrabbit
+                    url: "http://b"
+                username: ~
+                workspace: website
+                password: "root"
+
+    odm:
+        auto_generate_proxy_classes: true
+        document_managers:
+            default:
+                session: default
+                mappings:
+                    SandboxMainBundle: ~
+
+            website:
+                session: website
+                configuration_id: sandbox_magnolia.odm_configuration
+                mappings:
+                    SandboxMagnoliaBundle: ~
```

Comment:
I think I would be tempted to remove these tests. Adding this implies that we should add this much code for each configuration option, that would become increasingly difficult to maintain.

If we test that the configuration loads (which we do currently) and add the `->isRequired` flag then thats good enough imo.